tests/double_tests.c

```diff
@@ -12,3 +12,38 @@ Ensure(Double, comparison_with_negative_values_should_not_always_succeed) {
     /*  Issue #146 - Fixed */
     assert_that_double(-500, is_not_equal_to_double(0));
 }
+
+xEnsure(Double, relative_epsilon_calculations_with_0_and_a_very_small_number_should_not_fail) {
+    /* Issue #147
+
+    DM: I don't think this test case is valid. We should delete it.
+
```

Comment:
This is essentially rehashing the discussion from #147. I propose that we remove this test case as it is expected behavior given the way `significant_figures` works. I can fix up the pull request depending on our decision.